calpoly-csai/argo-editor
# Midas Depth Estimation MVP

Proof of concept of Midas depth estimation using the TFLite setup put together by @dahlgreb.
## Goal
Get an image depth estimation passing to and from the UI
## Testing
1. On this branch run `npm start`
2. Upload an image using the (ugly but functional) file upload button
3. Click on the find depth button
4. Scroll down to see the depth estimation image

I've tried this with pngs and jpegs. Lmk if you run into any serious issues.
